### DISCUSSION TITLE
Sesotho Model development

### DISCUSSION THREAD

**Discussion by AtomLaw** [explosion/spaCy#6617] [Language Support]
Greetings,
I would like to enquire on developing language models in Sesotho.
I saw the existing model, based on some of the samples observed, I have come to the realisation that our language and the South African version of Sesotho seem to be significantly different. So if possible even though I'm a beginner, I would like to participate with guidance in developing the model.

Sincerly

**Comment by yosiasz** [comment]
@AtomLaw are there 2 separate iso language codes for Sesotho? You say "our language", what country are you from?

I see 2 isos for that language (north and south)
nso
sot

I think it would help you to first identity which iso represents "your language". But if there is really only 1 iso and just regional variations, you might have to include those variations in there. I don't think one can claim theirs as the "correct" version. You might have to share 😉 the space, language does not know borders, it is as fluid as humans. I did see someone that posted something about the Romanian language where there are accepted different styles of spelling the same word. You might have to do the same.

Can you provide some examples of the differences you mention?

**Reply by AtomLaw** [reply]
Thank you,
Im from Lesotho, so given that there is a difference in how Sesotho is being used in Lesotho and South Africa, there tends to be a distinct difference.
Lesotho's ISO code is LS, Given that it's something that I'm attempting it would be a first. However, when it comes to Lesotho there are some other languages that need to recognised, then that's another task on its own.

I'll make a point to gather a corpus for this, but Ill make a point to send you some of the examples.

**Reply by yosiasz** [reply]
LS is iso country code for Lesotho, nothing to do with language?
